### PR TITLE
locksmithctl: add etcd authentication

### PR DESCRIPTION
**Pull request by jipperinbham**
expose username and password as options for configuration to etcd

### CODE REVIEW

File: locksmithctl/locksmithctl.go [hunk 1]
```diff
@@ -215,6 +219,10 @@ func getClient() (*lock.EtcdLockClient, error) {
 		Endpoints: globalFlags.Endpoints,
 		Transport: transport,
 	}
+	if globalFlags.EtcdUsername != "" && globalFlags.EtcdPassword != "" {
```
Comment (crawford) [inline locksmithctl/locksmithctl.go]: Why gate the assignment on this condition?

Comment (jipperinbham) [inline locksmithctl/locksmithctl.go]: good point, looking at https://github.com/coreos/locksmith/blob/master/Godeps/_workspace/src/github.com/coreos/etcd/client/client.go#L181-L186 it's not necessary.

**Comment by crawford** [comment]
LGTM. Go ahead and squash.

**Comment by crawford** [comment]
Thanks!

**Pull request merged**